core/kazoo_number_manager/src/knm_number_states.erl

```diff
@@ -55,7 +55,7 @@ change_state(Number, _State) ->
 to_aging(Number) ->
     to_aging(Number, number_state(Number)).
 
-to_aging(Number, ?NUMBER_STATE_AGING) ->
+to_aging(Number, ?NUMBER_STATE_AVAILABLE) ->
```

Comment:
i think this is another typo, i think it should be `NUMBER_STATE_RELEASED`, but @fenollp would know better.